### PR TITLE
Increase height of "add" buttons

### PR DESCRIPTION
**Pull request by rmcintosh**
From:
screen shot 2017-05-04 at 11 29 53 am
To:
screen shot 2017-05-04 at 11 29 49 am

### CODE REVIEW

**Comment by caker** [comment]
+1

File: scss/layout/index.scss [hunk 1]
```diff
@@ -1,6 +1,9 @@
 @import 'Layout';
 
 .linode-add {
+    height: 40px;
+    line-height: 26px;
```
Comment (eatonphil) [inline scss/layout/index.scss]: Could we do:

```
padding: 10px 15px;
height: auto;
```

Instead? This keeps a little better with how the other buttons work. And it gives it some extra left-right padding too.

Comment (rmcintosh) [inline scss/layout/index.scss]: works for me!

**Comment by eatonphil** [comment]
Now that I see it, @mparke, it's weird we have padding set on these buttons and height set too...

**Comment by eatonphil** [comment]
thank you!

**Pull request merged**